src/memex/views.py

```diff
@@ -82,7 +99,41 @@ def api_config(**settings):
         request_method = ('DELETE', 'GET', 'HEAD', 'POST', 'PUT',)
     settings['request_method'] = request_method + ('OPTIONS',)
 
-    return view_config(**settings)
+    if link_name:
+        link = {'name': link_name,
+                'method': initial_request_method,
```

Comment:
That being said, it might make sense to add an integration test for `GET /api` and make sure that the JSON data types are as expected (no array for `method`, as an example).